cookiecutter.json

```diff
@@ -0,0 +1,5 @@
+{
+  "project_name": "Project Name",
+  "company_name": "thoughtbot",
+  "bundle_identifier": "com.{{ cookiecutter.company_name | replace(' ', '-') | lower }}"
```

Comment:
I was thinking about that. Maybe just removing spaces? I think we can do that in a separate PR if we determine to go down this road further.